tests/entities_item_tests.py

```diff
@@ -202,10 +203,80 @@ def test_item_should_get_column_value_with_extra_id(get_column_values, get_items
     eq_(column_value.value, 'Hello, Grandma')
     eq_(type(column_value), cv.LongTextValue)
 
+@patch('moncli.api_v2.get_items')
+@patch.object(en.Item, 'get_board')
+@raises(ItemError)
+def test_should_raise_error_when_id_and_title_and_column_value_are_none(get_board,get_items):
+    # Arrange
+    get_items.return_value = [{'id': '1', 'name': 'Test Item 1'}]
+    get_board.return_value = en.Board(creds=None, id='1', name='Test Board 1')
+    item = client.get_items()[0]
+
+    # Act
+    item.change_column_value(id=None,title=None,column_value = None)
+
+@patch('moncli.api_v2.get_items')
+@patch.object(en.Item, 'get_board')
+@raises(ItemError)
+def test_should_raise_error_when_id_and_title_are_not_none(get_board,get_items):
+    # Arrange
+    get_items.return_value = [{'id': '1', 'name': 'Test Item 1'}]
+    get_board.return_value = en.Board(creds=None, id='1', name='Test Board 1')
+    item = client.get_items()[0]
+    id = "item_id"
+    title = "item_title"
+    # Act
+    item.change_column_value(id=id,title=title)
+
+@patch('moncli.api_v2.get_items')
+@raises(ItemError)
+def test_should_raise_item_error_for_invalid_column_value(get_items):
+    # Arrange
+    get_items.return_value = [{'id': '1', 'name': 'Test Item 1'}]
+    item = client.get_items()[0]
+    id = "item_id"
+    title = "item_title"
+    # Act
+    item.change_column_value(id=id,title=title,column_value=5)
 
+@patch.object(en.Item,'change_column_value')
```

Comment:
Dont mock the method call from the Item object itself. That defeats the purpose of the test.

Mock the moncli.api_v2.change_column_value method.